### ISSUE TITLE
exe from bdist_msi doesn't run as intended on devices other than mine

### ISSUE THREAD

**Issue by karimawi** [marcelotduarte/cx_Freeze#1740]
**Describe the bug**
I don't know if it's even a bug, I might be doing smth wrong, but I genuinely have no idea what's wrong

**To Reproduce**
I have a python file (guava.py) that I freeze with cx_Freeze, using this setup.py

```python
from cx_Freeze import setup, Executable

packs = ["tkinter","subprocess","notifypy","wget","requests","urllib","ctypes","os","threading","time","webbrowser","easygui","shutil","pyperclip","ahk","AppOpener","tinydb","tkextrafont","ahk_binary"]

setup(
    name= "Guava", options = {"build.exe":{"packages":packs,"includes":packs,"excludes":["PyQt5","PIL"]}},
    version = "1.0.0",
    description = "Guava",
    executables = [Executable('guava.py',icon='guavaicon.ico',
    shortcut_name="Guava",
    shortcut_dir="DesktopFolder",
    base="Win32GUI")]
)
#command:
#python setup.py bdist_msi --install-icon E:\Karim\Media\Code\Guava\GuavaFreeze\guavaicon.ico
```

here's my [script](https://github.com/karimawii/Guava/blob/master/guava.py)

Hey, I just wanna say, everything is working fine when I launch this on my device, this happens only on devices other than mine

I was using cx_Freeze 6.13.1, I got [this](https://user-images.githubusercontent.com/69881381/210430481-94ef602e-a17c-4ce9-86ef-10f10cccc082.png)
I then changed the setup.py a little bit to include urllib and requests but still got the same error
Then I tried using 6.14.0.dev1, I then got [this](https://user-images.githubusercontent.com/69881381/210430816-b51f7fa0-77a3-4d37-a24d-9d53c2693f95.png)
So I installed ahk_binary and added it to the setup.py + imports
then freezed again, and got [this](https://user-images.githubusercontent.com/69881381/210431003-c40517a2-8d8b-489b-93fb-13d98fc5fb09.png) which is the same as the first one.

The paths in those errors all have my Windows username, which is not the username of the device I'm on, so that's kinda weird, and I literally have no idea why this happens, it works fine on my end, the script itself runs no problem, I don't get it.

**Desktop (please complete the following information):**
 - OS :  Windows 10 64-bit
 - OS architecture : amd64
 - cx_Freeze version : (mentioned above)
 - Python version [e.g. 3.10]:

my device is also the same specs

**Additional context**
I know my code is not the best, I know I might be doing smth very wrong, but I clearly need some help.

**Comment by karimawi** [comment]
I used `"replace_paths": [("*", "")]` in the options and now it just shows the AHK error, idk how to include ahk in the installation directory, I tried `"include_files":[r"E:\Karim\Media\Code\Guava\GuavaFreeze\AutoHotkey\AutoHotkey.exe"]` but it doesn't copy it to the installation directory and still gives the same error

**Comment by marcelotduarte** [comment]
[include_files](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-include_files) accepts 2-tuples for source and destination, so you can try to use [(file, "lib/"+file)] or to lib/module/file.

**Comment by karimawi** [comment]
I tried `"include_files":[("E:/Karim/Media/Code/Guava/GuavaFreeze/AutoHotkey/AutoHotkey.exe","lib/AutoHotKey.exe")]` and it still didn't copy it in the install folder when installing, I remove the whole folder every time I freeze and install. I also tried `"include_files":[("E:/Karim/Media/Code/Guava/GuavaFreeze/AutoHotkey/AutoHotkey.exe","AutoHotKey.exe")]` , it's just not copying anything, I even used wget to download data I wanted to include from urls because it doesn't work, also the directory is right, the file is there, idk why it's not working

**Comment by marcelotduarte** [comment]
Does the file "E:/Karim/Media/Code/Guava/GuavaFreeze/AutoHotkey/AutoHotkey.exe" exist before you run the setup?

**Comment by karimawi** [comment]
yes, just to test if it'll work, I used wget with the ahk exe, and it worked, didn't show any errors, so that's good, I just need to know why include_files isn't working

**Comment by karimawi** [comment]
here you go:
![Code_rZH7edOdku](https://user-images.githubusercontent.com/69881381/210900503-5b2929cc-dc9a-43bf-8b92-f896bcb1a60d.gif)

**Comment by marcelotduarte** [comment]
@karimawii Checking the [error](https://user-images.githubusercontent.com/69881381/210430816-b51f7fa0-77a3-4d37-a24d-9d53c2693f95.png)
I can suggest you to use AHK(executable_path='C:\\path\\to\\AutoHotkey.exe') as documented [here](https://github.com/spyoungtech/ahk#non-python-dependencies)
If you use, in setup, "include_files":[r"E:\Karim\Media\Code\Guava\GuavaFreeze\AutoHotkey\AutoHotkey.exe"]
Basically you should change line 158/159 to:
```python
if getattr(sys, "frozen", False):
    # The application is frozen
    ahk = AHK(executable_path=os.path.join(sys.frozen_dir, "AutoHotkey.exe"))
else:
    ahk = AHK()
```

**Comment by karimawi** [comment]
I did try exactly that, didn't work, but, I'm replacing AHK with multithreaded PyAutoGUI, and it will have a smaller part since I'll use network-based login instead of fully relying on screen capture, I'll try freezing with cx_Freeze after finalizing the code for that, if it didn't work I'll reopen this or just build it with an alternative library, thinking of NSIS for the installer part, I'll just zip everything and use the "exe installer from zip" function in NSIS, as for the compiling of the actual Python script I'm thinking of either trying the exe compiler in cx_Freeze, see if that works for me, or use Pyinstaller, let me know if you have better ideas, I'm going to close this issue for now, and I'll post any related updates once there's any.
